bumble/device.py

```diff
@@ -479,8 +478,8 @@ def request_pairing(self):
     async def authenticate(self):
         return await self.device.authenticate(self)
 
-    async def encrypt(self):
-        return await self.device.encrypt(self)
+    async def encrypt(self, enable=0x01):
```

Comment:
suggest: make `enable` a boolean, and only map it to the constant 0x01 at the lower level (HCI)